### PR TITLE
Feat/msdk 335 portfolio custodian role

### PR DESCRIPTION
**Pull request by shuffledex**

### CODE REVIEW

File: src/api/procedures/setCustodian.ts [hunk 1]
```diff
@@ -43,17 +43,7 @@ export async function prepareSetCustodian(
...
-    portfolio.getCustodian(),
-  ]);
-
-  if (custodian.did !== currentIdentity.did) {
-    throw new PolymeshError({
-      code: ErrorCode.ValidationError,
-      message: 'You are not the custodian of this portfolio',
-    });
-  }
+  const currentIdentity = await context.getCurrentIdentity();
```
Comment (monitz87) [inline src/api/procedures/setCustodian.ts:46]: This can now be requested alongside authorizationRequests in a Promise.all

**Review by monitz87 — CHANGES_REQUESTED** [review]

File: src/api/procedures/moveFunds.ts [hunk 2]
```diff
@@ -73,20 +73,9 @@ export async function prepareMoveFunds(this: Procedure<Params, void>, args: Para
...
-      tokens: items.map(({ token }) => token),
-    }),
-  ]);
-
-  if (!toIsOwned) {
-    throw new PolymeshError({
-      code: ErrorCode.ValidationError,
-      message: 'You must be the owner of both Portfolios',
-    });
-  }
```
Comment (monitz87) [inline src/api/procedures/moveFunds.ts]: This is not what was requested

**Review by monitz87 — CHANGES_REQUESTED** [review]

File: src/api/procedures/moveFunds.ts (hunk 2 above)
Comment (shuffledex) [reply src/api/procedures/moveFunds.ts]: the validation you requested is already implemented in line 62

Comment (monitz87) [reply src/api/procedures/moveFunds.ts]: Absolutely right. I'm sorry, my bad

**Review by monitz87 — APPROVED** [review]

**Pull request merged**